libs/features/icons/icons.js

```diff
@@ -34,7 +34,17 @@ export const fetchIcons = (config) => new Promise(async (resolve) => {
   resolve(fetchedIcons);
 });
 
-function decorateToolTip(icon) {
+function checkIfIconIsTooltip(icon) {
+  const wrapper = icon.closest('em');
+  if (!wrapper) return null;
+
+  const conf = wrapper.textContent.includes('|');
+  if (!conf) return null;
+
+  return [...icon.classList].some((cls) => cls.includes('tooltip'));
+}
```

Comment:
You can shorten this a bit, in fact if you shorten this, you can probably get rid of this helper function
```suggestion
function checkIfIconIsTooltip(icon) {
  const hasAuthoredTooltipText = icon.closest('em')?.textContent.includes('|');
  return hasAuthoredTooltipText && [...icon.classList].some((cls) => cls.includes('tooltip'));
}
```